modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -92,7 +116,12 @@ installation, do not delete these files.
 . Configure the network boot infrastructure so that the machines boot from their
 local disks after {op-system} is installed on them.
 
-. Configure PXE or iPXE installation for the {op-system} images and begin the installation.
+ifndef::only-pxe[]
+. Configure PXE or iPXE installation for the {op-system} images.
+endif::only-pxe[]
+ifdef::only-pxe[]
+. Configure PXE installation for the {op-system} images.
+endif::only-pxe[]
 +
 Modify one of the following example menu entries for your environment and verify
```

Comment:
This line states "one of the following example menu entries". In the IBM Power assemblies in the preview for this PR, there is only one example, because the iPXE one is not present. You could make this line an `ifndef::only-pxe[]` conditional and then create an ifdef::only-pxe[]` conditional that provides an alternative line that states something like the following:

"Modify the following example menu entry for your environment and verify that the image and Ignition files are properly accessible:"

Reply:
I can't comment directly on line 129 in this PR, but the same applies to the `** For PXE:` bullet point. You could make it an `ifndef::only-pxe[]` conditional because the whole module is only about PXE for IBM Power.